Tetris/src/tetris/SystemTester.java

```diff
@@ -0,0 +1,64 @@
+package tetris;
+
+public class SystemTester {
```

Comment:
Hopefully this reply will post to the group!

Hi Michelle,

The SystemTester.java is great! It really helps show what is going to be 
happening with they whole system.

I recognize that collision detection isn't working yet.

I've noticed that when the new tetromino first appears that only some of 
it is showing and when the tetromino lands that the highest row of the 
tetromino disappears.

I added a couple of things to the SystemTester to rotate and shift the 
tetromino. Rotation appears good but the error appears to be in my line 
that is rotating the tetromino. except that if I comment out the shift 
left / right then the problem goes away. So...?

    Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 10
         at tetris.GameSystem.moveActiveTetromino(GameSystem.java:184)
         at tetris.SystemTester.main(SystemTester.java:67)

*/_IF_/* I have done my git correctly the SystemTester with my updates 
should be in my remote repository.

Call me if it isn't or you have questions
(408) 390-5365

Thanks!
-Shawn

On 8/19/16 9:43 PM, Michelle Soult wrote:
>
> In Tetris/src/tetris/SystemTester.java 
> <https://github.com/miso8263/COEN275/pull/8#discussion_r75572643>:
>
> > @@ -0,0 +1,64 @@
> > +package tetris;
> > +
> > +public class SystemTester {
>
> @vonyar <https://github.com/vonyar> Here is the system tester
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/miso8263/COEN275/pull/8/files/384a28f312ae1b384a94857a350772b03d8c3fbd#r75572643>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AKHSdqDFpxMnXldLwb2OyyJzOBHDsUSSks5qhoXdgaJpZM4Jo_vS>.
>